tsconfig.json

```diff
@@ -10,7 +10,9 @@
     "strict": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "moduleResolution": "node"
+    "moduleResolution": "node",
+    "skipLibCheck": true
```

Comment:
What does this do?

Reply:
Skips checking node modules type definitions related to your code.

Reply:
Why do we need it now when we didn't before?

Reply:
We have more deps now that are poorly typed. before the compile didn't fail when checking libs now it does.